### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.10 es 4.75 (13 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#325] [Esercizi M-Z]
![immagine](https://github.com/user-attachments/assets/a0444933-31ff-42de-86d0-ca1a42d3538f)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/09765eca-18cd-48a5-a8ea-ce72b4e7cda5)
